File: Models/Model/TsneCoordinate.cs
using System;
using System.Collections.Generic;

namespace WorldBT.Models.Model
{
    public partial class TsneCoordinate
    {
        public Guid Id { get; set; }
        public Guid PatientId { get; set; }
        public decimal X { get; set; }
        public decimal Y { get; set; }

        public virtual Patient Patient { get; set; }
    }
}

File: Web/src/app/models/tsneCoordinate.ts
import { Patient } from "./patient";

export class TsneCoordinate {
    Id: string;
    X: number;
    Y: number;
    Patient: Patient;

    constructor(
        id: string = "",
        x: number = 0,
        y: number = 0
    ) {
        this.Id = id,
        this.X = x,
        this.Y = y
    }
}